LandSandBoat/server
# [SMN] Correct Shiva animations & Blood Pact abilities

**_I affirm:_**

## What does this pull request do?



Updates all of Shiva's animations to reflect new pet_skills sql table
Compared all Blood pact msg's to retail captures
Removes Shiva's animation shifting from core
Added SQL updates for future Shiva Blood Pact abilities

## Steps to test these changes


Use Shiva's BP Ward to see proper MP consumption and messages displayed
Use Shiva's BP Rage to see proper MP consumption and messages displayed
